app/src/main/java/vn/ztech/software/projectgutenberg/screen/home/ListBookAdapter.kt

```diff
@@ -0,0 +1,65 @@
+package vn.ztech.software.projectgutenberg.screen.home
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import vn.ztech.software.projectgutenberg.data.model.Book
+import vn.ztech.software.projectgutenberg.databinding.ItemBookBinding
+import vn.ztech.software.projectgutenberg.utils.extension.findCoverImageURL
+import vn.ztech.software.projectgutenberg.utils.extension.findShowableAgent
+
+class ListBookAdapter(private val listener: OnClickListener) :
```

Comment:
suggesst a zai dùng hight order function 
```suggestion
class ListBookAdapter(private val onClick: (book : Book)->Unit) :
```